strongbox-security/strongbox-security-api/src/main/java/org/carlspring/strongbox/configuration/WebSecurityConfig.java

```diff
@@ -12,14 +12,14 @@
 @ComponentScan({ "org.carlspring.strongbox.configuration",
                  "org.carlspring.strongbox.security",
                  "org.carlspring.strongbox.visitors" })
-public class StrongboxSecurityConfig
+public class WebSecurityConfig
```

Comment:
seems we have some confusion with this naming refactoring, details [here](https://github.com/strongbox/strongbox/issues/1840#issuecomment-709946439)

Reply:
@madhukarbharti please leave `StrongboxSecurityConfig` name here and rename the `/strongbox-web-core/src/main/java/org/carlspring/strongbox/config/SecurityConfig.java` to `WebSecurityConfig` instead.